AdblockPlusSafari/AppDelegate.m

```diff
@@ -20,14 +20,27 @@
 #import "Appearence.h"
 #import "RootController.h"
 
+// Update filter list every 5 days
+const NSTimeInterval FilterlistsUpdatePeriod = 3600*24*5;
+// Wake up application every 12 hours (just hint for iOS)
+const NSTimeInterval BackgroundFetchInterval = 3600*12;
```

Comment:
Every 12 hours starting when? The first start of the app? Is it technically possible that the time between checks will be more than 12 hours?

It might be better to go for a shorter interval here, we use one hour in the ABP core code.